app/components/ui/set-up-domain/progress-header/index.js

```diff
@@ -0,0 +1,37 @@
+// External dependencies
+import i18n from 'i18n-calypso';
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+
+// Internal dependencies
+import { getPath } from 'routes';
+import ProgressBar from 'components/ui/progress-bar';
+import styles from './styles.scss';
+
+const ProgressHeader = ( { children, progress } ) => {
+	return (
+		<div className={ styles.headerContainer }>
+			<div className={ styles.header }>
+				<h1 className={ styles.headerText }>
+					<span className={ styles.setUpLabel }>
+						{ i18n.translate( 'Setup: ' ) }
+					</span>
+					{ children }
+				</h1>
+				<div className={ styles.progressHeader }>
+					<div className={ styles.fullWidth }>
+						<ProgressBar progress={ progress } />
+					</div>
+					<a className={ styles.exit } href={ getPath( 'myDomains' ) }>{ i18n.translate( 'Exit' ) }</a>
```

Comment:
@aidvu Sorry, I gave you bad advice here! We need to use the `Link` element for internal links, otherwise the page reloads when the user clicks this.

```js
import { Link } from 'react-router'; 

<Link to={ getPath( 'myDomains' ) }>...</Link>
```